zero_bounce_sdk/src/main/java/com/zerobounce/android/ZeroBounceSDK.kt

```diff
@@ -147,6 +156,75 @@ object ZeroBounceSDK {
         )
     }
 
+    /**
+     * Find the email based on a given [firstName], [domain] or [companyName].
+     *
+     */
```

Comment:
Please provide extensive documentation similar to the existing one.

```suggestion
    /**
     * Finds the email based on a given [firstName], [domain] or [companyName].
     *
     * **Note**: At least one of [domain] or [companyName] must be provided.
     *
     * @param firstName the first name of the person whose email format is being searched
     * @param domain the email domain for which to find the email format
     * @param companyName the company name for which to find the email format
     * @param middleName the middle name of the person whose email format is being searched;
     * optional
     * @param lastName the last name of the person whose email format is being searched; optional
     * @param responseCallback the response callback
     * @param errorCallback the error callback
     */
```